extensions/adobe/experience/analytics/experienceevent.schema.json

```diff
@@ -7212,6 +7212,18 @@
               "description": "Custom event 1000."
             }
           }
+        },
+        "https://ns.adobe.com/experience/analytics/environment/browserID": {
+          "title": "Browser ID",
+          "type": "integer",
+          "description":
+            "The Adobe Analytics Identifier for the browser used."
+        },
+        "https://ns.adobe.com/experience/analytics/environment/operatingSystemID": {
```

Comment:
Should be just `Should be just `https://ns.adobe.com/experience/analytics/operatingSystemID `